doc/developers-guide/SCRAM-serialization.md

```diff
@@ -0,0 +1,22 @@
+## Overview
+
+This document describes the SCRAM serialization format used by MongooseIM.
+Developers can use this information to create advanced endpoints for `ejabberd_auth_http` or enable other software to read (i.e. share) the user authentication data.
```

Comment:
read (i.e. share) user authentication data.

Reply:
I would keep the "the":)